source/mdcii/mdcii/include/mdcii/gam/doc/scene.md

```diff
@@ -0,0 +1,141 @@
+|              |        | TabacoPlantation | 1336  |
+|              |        | WinePlantation   | 1344  |
+|              |        | SugarPlantation  | 1340  |
+|              |        | CacaoPlantation  | 1338  |
+|              |        | WoolPlantation   | 1332  |
+|              |        | SpicesPlantation | 1342  |
+
+### Structs
+
+| RandomGood | Type         | Description                               |
+| ---------- | ------------ | ----------------------------------------- |
+| houseId    | GoodsHouseId | specifies the houseId that can build this |
+| amount     | uint16       | specifies how many resources there are    |
+| -          | union uint32 | unused? kind, price, percent              |
+
+| RandomOre    | Type       | Description                          |
+| ------------ | ---------- | ------------------------------------ |
+| smallOreMine | RandomGood | specifies the random small ore mines |
+| greatOreMine | RandomGood | specifies the random great ore mines |
+| goldMine     | RandomGood | specifies the random gold mines      |
+| empty        | RandomGood | unused                               |
+
+| RandomRawMaterials | Type          | Description                                                    |
+| ------------------ | ------------- | -------------------------------------------------------------- |
+| tabaco             | RandomGood    | specifies the distribution for random tabaco 100% growth rates |
+| spices             | RandomGood    | specifies the distribution for random spices 100% growth rates |
+| sugar              | RandomGood    | specifies the distribution for random sugar 100% growth rates  |
+| wool               | RandomGood    | specifies the distribution for random wool 100% growth rates   |
+| wine               | RandomGood    | specifies the distribution for random wine 100% growth rates   |
+| cacao              | RandomGood    | specifies the distribution for random cacao 100% growth rates  |
+| empty              | RandomGood[6] | empty                                                          |
+
+| RandomGoodies | Type          | Description                          |
+| ------------- | ------------- | ------------------------------------ |
+| treasure      | RandomGood    | specifies amount of random treasures |
+| empty         | RandomGood[3] | empty                                |
+
+| RandomHardware | Type          | Description |
+| -------------- | ------------- | ----------- |
+| empty          | RandomGood[8] | unused?     |
+
+| RandomNativeVillages | Type      | Description                                |
+| -------------------- | --------- | ------------------------------------------ |
+| strawRoofCount       | uint32    | specifies the count of straw roof villages |
+| incasCount           | uint32    | specifies the count of incas villages      |
+| empty                | uint32[3] | empty                                      |
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/armin.schlegel/mdcii-engine/pullRequest?prid=5498758)